backend/zfs_operations/services/dataset_service.py

```
@@ -0,0 +1,576 @@
+from typing import List, Optional, Dict, Any
+from datetime import datetime
+import logging
```

Ruff (0.11.9)

`logging` imported but unused

Remove unused import: `logging`

(F401)